Source/Core/DolphinQt/DolphinQt.vcxproj

```diff
@@ -488,6 +488,9 @@
     <ProjectReference Include="$(ExternalsDir)Bochs_disasm\Bochs_disasm.vcxproj">
       <Project>{8ada04d7-6db1-4da4-ab55-64fb12a0997b}</Project>
     </ProjectReference>
+    <ProjectReference Include="$(ExternalsDir)imgui\imgui.vcxproj">
```

Comment:
Everything else has `CoreDir` first and then `ExternalsDir`, any particular reason why this one is sorted differently?

They _are_ sorted alphabetically within their respective groups though, so maybe we'll just take it like that?

Reply:
thanks, fixed